nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RGroupIT.java

```diff
@@ -57,7 +58,8 @@
   public static Option[] configureNexus() {
     return NexusPaxExamSupport.options(
         NexusITSupport.configureNexusBase(),
-        nexusFeature("org.sonatype.nexus.plugins", "nexus-repository-r")
+        nexusFeature("org.sonatype.nexus.plugins", "nexus-repository-r"),
+        editConfigurationFileExtend(NEXUS_PROPERTIES_FILE, "nexus.r.packagesBuilder.interval", "1")
```

Comment:
Make "1" and line 114 related from each other to decrease misunderstanding.
Ex. Integer interval = 1L => "1" = interval.toString();  Thread.sleep(interval * 1000);

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27